manifests/Main Plot Plus.yaml

```
@@ -0,0 +1,16 @@
+name: Main Plot Plus
+authors: ziproot
+homepage: https://github.com/ziproot/endless-sky-main-plot-plus
+license: GPL-3+
```

```suggestion
license: GPL-3.0-or-later
```
[GPL-3+ is deprecated](https://spdx.org/licenses/GPL-3.0+.html)

---

ninja'd